cooldowns_mage.lua

```diff
@@ -55,6 +55,26 @@ LCT_SpellData[80353] = {
 	duration = 40,
 	cooldown = 300
 }
+
+-- Mage/mixed
+-- Alter Time (fire & frost)
+LCT_SpellData[108978] = {
+	class = "MAGE",
+	specID = { SPEC_MAGE_FIRE, SPEC_MAGE_FROST },
+	defensive = true,
+	duration = 10,
+	cooldown = 60
+}
+-- Alter Time (arcane)
+LCT_SpellData[342245] = {
+	class = "MAGE",
+	specID = { SPEC_MAGE_ARCANE },
+	defensive = true,
+	duration = 10,
+	cooldown = 60,
+	opt_lower_cooldown = 30, -- with 342249 Master of Time

```

Comment:
This one I'm not so confident, since if the player picks Master of Time, it also means Alter Time should reset the cooldown of Blink/Shimmer.
I'm not sure how to add that info here

Reply:
I don't think there's a way to do this sadly. It's doable if it's a different spell id. But here, it also only gives a charge back if you don't cancel alter time, so it's double unsupported.

Reply:
Hmm I see.
Do you think it's better to just not support Arcane Alter Time for now, since it will mess up the Blink/Shimmer cooldowns?
I don't think Master of Time is played that much in 3s but in double dps 2s it's quite possible.

Reply:
>  Do you think it's better to just not support Arcane Alter Time for now, since it will mess up the Blink/Shimmer cooldowns?

The cooldowns for shimmer/blink will have to stay unreliable for now.